chef/data_bags/crowbar/migrate/ipmi/101_bmc_interface.rb

```diff
@@ -0,0 +1,11 @@
+def upgrade(ta, td, a, d)
+  unless a["ipmi"].key? "bmc_interface"
+    a["ipmi"]["bmc_interface"] = ta["ipmi"]["bmc_interface"]
```

Comment:
Are you sure this is right? I'm getting a migration error, it looks like it might be just `a.key? "bmc_interface"` and `a["bmc_interface"] = ta["bmc_interface"]`